packages/edit-site/src/components/sidebar-edit-mode/template-panel/style.scss

```diff
@@ -54,4 +54,8 @@ h3.edit-site-template-card__template-areas-title {
 	@include break-wide() {
 		column-count: 4;
 	}
+
+	.block-editor-block-patterns-list__list-item {
+		break-inside: avoid-column;
```

Comment:
We have the same css rule in `swap template` modal and while it handles the current use case that it fixes, it does have the problem @Mamaduka describes. I tried a few things but didn't figure out a proper solution. Maybe we can achieve the same result with a different css rule? 🤔 --cc @jameskoster @jasmussen